tests/HTTP/ClientTest.php

```diff
@@ -34,6 +34,38 @@ public function testCreateCurlSettingsArrayGET(): void
         self::assertEquals($settings, $client->createCurlSettingsArray($request));
     }
 
+    public function testCreateCurlSettingsHTTPHeader(): void
+    {
+        $client = new ClientMock();
+        $header = [
+            'Authorization: Bearer 12345'
+        ];
+        $client->addCurlSetting(CURLOPT_POSTREDIR, 0);
+        $client->addCurlSetting(CURLOPT_HTTPHEADER, $header);
+
+        $request = new Request('GET', 'http://example.org/');
+
+        $settings = [
+            CURLOPT_RETURNTRANSFER => true,
+            CURLOPT_HEADER => true,
+            CURLOPT_POSTREDIR => 0,
+            CURLOPT_HTTPHEADER => ['Authorization: Bearer 12345'],
+            CURLOPT_NOBODY => false,
+            CURLOPT_URL => 'http://example.org/',
+            CURLOPT_CUSTOMREQUEST => 'GET',
+            CURLOPT_USERAGENT => 'sabre-http/'.Version::VERSION.' (http://sabre.io/)',
+        ];
+
+        // FIXME: CURLOPT_PROTOCOLS and CURLOPT_REDIR_PROTOCOLS are currently unsupported by HHVM
+        // at least if this unit test fails in the future we know it is :)
+        if (false === defined('HHVM_VERSION')) {
```

Comment:
do we still run HHVM in CI? if so, we should drop it in a separate PR, so this If-case is not needed.

HHVM is not worth supporting

Reply:
No, we don't do HHVM any more- long gone. So copy-pasting that stuff into new tests is not needed.
But I can do a cleanup after this PR.